DDG4/src/Geant4GeneratorActionInit.cpp

```diff
@@ -30,6 +30,8 @@ Geant4GeneratorActionInit::Geant4GeneratorActionInit(Geant4Context* ctxt, const
   InstanceCount::increment(this);
   context()->kernel().runAction().callAtEnd(this,&Geant4GeneratorActionInit::end);
   context()->kernel().runAction().callAtBegin(this,&Geant4GeneratorActionInit::begin);
+  declareProperty("numberOfEvents", m_evtTotal);
```

Comment:
@MarkusFrankATcernch 
Is this the best way to let me read the number of events from the plugin, or is there something else that would be cleaner?